www/README.md

```diff
@@ -1,16 +1,51 @@
-# React documentation site
-
-To get setup:
-```
-yarn install
-```
-
-To run the dev server:
-```
-yarn run dev
-```
-
-To clear the local cache:
-```
-yarn run reset
-```
\ No newline at end of file
+# React — documentation site
+
+## Getting started
+
+### Prerequisites
+
+1. Git
+1. Node: install version 7.6 or greater
+1. Yarn: `npm i -g yarn` to install it globally via NPM
+1. A clone of the [https://github.com/facebook/react](React repo) on your local machine
+1. A fork of the repo (for any contributions)
+
+### Installation
+
+1. `cd www` to go into the website project
+1. `yarn` to install the website's NPM dependencies
+
+### Running locally
+
+1. `yarn dev` to build the site with Gatsby, a (React-based!) static site generator
+1. `open http://localhost:8000` to open the site in your favourite browser
+
+## Contributing
+
+### Create a branch
+
+1. `git checkout master` from any folder in your local react repository
+1. `git pull origin master` to ensure you have the latest main code
+1. `git checkout -b wwwTheNameOfMyChange` (replacing `TheNameOfMyChange` with a suitable name) to create a branch
+
+### Make the change
+
+1. Follow the "Running locally" instructions
+1. Save the files and check in the browser (any React components in `src/components` or `src/templates`) will hot-reload)
+
```

Comment:
Worth noting that if you change site internals like plugins, it may be necessary to remove `.cache` directory and restart the server for changes to take effect.

Reply:
Good idea. Split these out into different procedures